pkg/services/secrets/kvstore/migrations/migrator.go

```diff
@@ -80,6 +100,7 @@ func (s *SecretMigrationProviderImpl) Migrate(ctx context.Context) error {
 	})
 	if err != nil {
 		logger.Error("Server lock for secret migration already exists")
+		s.errs <- err // ????
```

Comment:
I'm 99.99% confident this is wrong - this error doesn't stop the service running (nothing does).